neural_lam/train_model.py

```diff
@@ -20,37 +24,45 @@
 }
 
 
+def _init_datastore(datastore_kind, config_path):
+    if datastore_kind == "multizarr":
+        datastore = MultiZarrDatastore(config_path=config_path)
+    elif datastore_kind == "npyfiles":
+        datastore = NpyFilesDatastore(config_path=config_path)
+    elif datastore_kind == "mllam":
+        datastore = MLLAMDatastore(config_path=config_path)
+    else:
+        raise ValueError(f"Unknown datastore kind: {datastore_kind}")
+    return datastore
+
+
 def main(input_args=None):
-    """
-    Main function for training and evaluating models
-    """
+    """Main function for training and evaluating models."""
     parser = ArgumentParser(
         description="Train or evaluate NeurWP models for LAM"
     )
     parser.add_argument(
-        "--data_config",
+        "datastore_kind",
         type=str,
-        default="neural_lam/data_config.yaml",
-        help="Path to data config file (default: neural_lam/data_config.yaml)",
+        choices=["multizarr", "npyfiles", "mllam"],
+        help="Kind of datastore to use",
+    )
+    parser.add_argument(
+        "datastore_config_path",
+        type=str,
+        help="Path for the datastore config",
```

Comment:
I'd strongly prefer to keep all argparse arguments as explicit flags rather than positional. Makes it clear what each option in a training script does. 
```suggestion
    parser.add_argument(
        "--datastore_config_path",
        type=str,
        help="Path for the datastore config",
```